hosted/eks/p1/p1_provisioning_test.go

```diff
@@ -168,6 +169,25 @@ var _ = Describe("P1Provisioning", func() {
 		Expect(amiID).To(Or(Equal("AL2_x86_64_GPU"), Equal("AL2023_x86_64_NVIDIA")))
 	})
 
+	It("Deploy a cluster with Public/Priv access then disable Public access", func() {
```

Comment:
```suggestion
	XIt("Deploy a cluster with Public/Priv access then disable Public access", func() {
```

Reply:
This is a flake. I think it might be okay to keep this, and if it fails, it fails :shrug: WDYT?

Reply:
I'd prefer to skip it since there's and an associated open bug and to turn suite run green :green_circle: :blush: